internal/cli/cmd/dac/build/build.go

```diff
@@ -218,6 +217,9 @@ percli dac build -d my_dashboards
 
 # build all the files under a given directory & deploy the resulting resources right away
 percli dac build -d my_dashboards && percli apply -d built
+
+# build a given file as JSON providing extra arguments to the Go program
+percli dac build -f main.go -ojson -- --arg1=value1 --arg2=value2
```

Comment:
perhaps we can have extra check to verify the extra args are following this struct ?

Reply:
@Nexucis sorry I'm afraid I didn't follow your idea, could you come again?

Reply:
I mean, does it makes sense to check that the syntax to catch the extra args start with `--` and then followed by `--arg1=value` ?

Reply:
I don't think so, because the structure of args will depend on the framework used to define flags on the rendered, which might be different between frameworks.

Wdyt @Nexucis ?